tests/phpunit/includes/testcase-preload-paths.php

```diff
@@ -163,15 +160,48 @@ protected function get_paths_dataset( $post ) {
 				'/wp/v2/types?context=edit',
 				'/wp/v2/taxonomies?per_page=-1&context=edit',
 				'/wp/v2/themes?status=active',
-				"/wp/v2/types/{$post->post_type}?context=edit",
-				"/wp/v2/users/me?post_type={$post->post_type}&context=edit",
-				"/wp/v2/{$post->post_type}s/{$post->ID}?context=edit",
-				"/wp/v2/{$post->post_type}s/{$post->ID}/autosaves?context=edit",
+				'/wp/v2/types/{post_type}?context=edit',
+				'/wp/v2/users/me?post_type={post_type}&context=edit',
+				'/wp/v2/{post_type}s/{ID}?context=edit',
+				'/wp/v2/{post_type}s/{ID}/autosaves?context=edit',
 				array(
 					0 => '/wp/v2/media',
 					1 => 'OPTIONS',
 				),
 			),
 		);
 	}
+
+	/**
+	 * Makes the data returned by preload_paths_provider concrete by creating a post and injecting its type and ID into the given path.
+	 *
+	 * @param string|string[] $path         The preload path under test. Could be an array if provided along a HTTP method.
+	 * @param bool            $is_filtered  Whether the path should be filtered or not.
+	 * @param string          $context_post Says what type of post must be created for the context. Possible values are `translatable`, `translatable_cpt`, `untranslatable`.
+	 * @return array                        The filtered path at key 0, the post at key 1, "is the post type translatable" at key 2.
```

Comment:
Maybe I didn't read correctly, but I didn't see where you use keys 1 and 2. Given that maybe we could simplify?

Reply:
They are used in `Preload_Paths_Test::test_preload_paths_with_post_editor_context()`.